examples/tests/hu.elte.txtuml.examples.machine/src/machine1/x/XMachine1Configuration.java

```diff
@@ -0,0 +1,10 @@
+package machine1.x;
+
+import hu.elte.txtuml.api.deployment.Group;
+import machine1.x.model.Machine;
+import machine1.x.model.User;
+
+@Group(contains = { Machine.class, User.class })
+public class XMachine1Configuration {
```

Comment:
Note that the marked file is in the _tests_ subfolder, not in _demo_. It seems this version was created in 224a896. Is it still necessary?